### PR TITLE
Add VsnipAdd command

### PR DESCRIPTION
**Pull request by jandamm**
This command quickly copies the selected lines and places them in the
clipboard to paste into the then opened json.

Relates to #65

### CODE REVIEW

**Comment by hrsh7th** [comment]
Thank you!
And sorry for the late reply.

I will check it!

**Comment by hrsh7th** [comment]
In my impression, It is useful but it has a lot of improvement points.

1. should fix indent as "\t"
2. the body should be multiline

I can merge it as the first implementation (with some fixes a bit)

File: plugin/vsnip.vim [hunk 1]
```diff
@@ -56,6 +56,17 @@ function! s:open_command(bang, cmd)
   \ )))
 endfunction
 
+command! -range -bang -nargs=? VsnipAdd call s:add_command(<bang>0, 'vsplit', <line1>, <line2>, <q-args>)
+function! s:add_command(bang, cmd, start, end, name) abort
+  let lines = map(getbufline('%', a:start, a:end), { key, val -> json_encode(substitute(val, '\$', '\\$', 'ge')) })
+	let format = "  \"VsnipAdd\": {\n    \"prefix\": [\"%s\"],\n    \"body\": [%s]\n  }"
```
Comment (hrsh7th) [inline plugin/vsnip.vim:62]: I think `VsnipAdd` key should be empty text.

**Comment by jandamm** [comment]
>     1. should fix indent as "\t"

Do you mean the `format` for `printf` should use tabs instead of spaces?

File: plugin/vsnip.vim (hunk 1 above)
Comment (jandamm) [reply plugin/vsnip.vim]: Don't giving it a name results in vsnip not working.
I've changed it to the same as the prefix value.

**Comment by jandamm** [comment]
@hrsh7th issues should be fixed now 👍

**Comment by hrsh7th** [comment]
Sorry!
I've consider it. I think It should be `VsnipYank`.
We can `:'<'>VsnipYank | VsnipOpen`.

**Comment by jandamm** [comment]
Done 👍 
You're now able to do `:VsnipYank key` to copy the current line and insert `key` as a keyword for the snippet 🙂

**Comment by hrsh7th** [comment]
LGTM. Thanks!

**Comment by hrsh7th** [comment]
This feature is useful. Thank you~!

**Pull request merged**